src/cr/cube/matrix.py

```diff
@@ -420,15 +424,28 @@ def __init__(self, dimensions, counts, base_counts, counts_with_missings=None):
     @lazyproperty
     def columns(self):
         return tuple(
-            _CategoricalVector(counts, base_counts, element, self.table_margin, zscore)
+            _CategoricalVector(
+                counts,
+                base_counts,
+                element,
+                self.table_margin,
+                zscore,
+                opposite_margins=np.sum(self._counts, axis=1),
```

Comment:
It's _easy_ to miss the `=1` here. I'd create a private property of `_rows_margin`, and just pass it here. Later, when we get to more complex cases with MR, this will come in handy.